spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/config/GcpConfigBootstrapConfigurationTest.java

```diff
@@ -51,15 +54,23 @@ public void testConfigurationValuesAreCorrectlyLoaded() {
 		this.contextRunner.withPropertyValues("spring.application.name=myapp",
 				"spring.profiles.active=prod",
 				"spring.cloud.gcp.config.timeoutMillis=120000",
-				"spring.cloud.gcp.config.enabled=false",
+				"spring.cloud.gcp.config.enabled=true",
 				"spring.cloud.gcp.config.project-id=pariah")
 				.run((context) -> {
 					GcpConfigProperties config = context.getBean(GcpConfigProperties.class);
 					assertThat(config.getName()).isEqualTo("myapp");
 					assertThat(config.getProfile()).isEqualTo("prod");
 					assertThat(config.getTimeoutMillis()).isEqualTo(120000);
-					assertThat(config.isEnabled()).isFalse();
+					assertThat(config.isEnabled()).isTrue();
 					assertThat(config.getProjectId()).isEqualTo("pariah");
 				});
 	}
+
+	@Test
+	public void testConfigurationDisabled() {
+		this.contextRunner.run((context) -> {
+			assertThatThrownBy(() -> context.getBean(GcpConfigProperties.class)).isInstanceOf(
```

Comment:
Not related and minor, but we recently opened for ourselves that `assertThatExceptionOfType()` is much handy for this kind of verifications: https://github.com/spring-projects/spring-integration/issues/2760